crates/catalog/glue/tests/glue_catalog_test.rs

```diff
@@ -92,6 +96,24 @@ async fn set_test_fixture(func: &str) -> TestFixture {
     }
 }
 
+async fn lazy_reuse_dc() -> Arc<TestFixture> {
```

Comment:
I have run it locally many times and found that most cases will be reused because these test cases are basically concurrent.

But if necessary, I will modify it to dtor. I have abstracted it into test_utils now, and it is very convenient to modify.